Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:latest
+
+# Install packages
+RUN apt update && apt install -y \
+    curl \
+    gnupg \
+    wget
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+ && apt-get install -y nodejs

```

Comment:
You could just use apt install. It would standardize on the apt command instead of having both apt-get and apt

Reply:
Unfortunately the version of node.js that is in the apt catalog is ancient -- something like three major versions old -- and it won't run the code in this repo.  This updates apt with the latest 8.x release of Node.

Reply:
I meant you can just use the apt command instead of apt-get. Apt is essentially a more modern command that is being moved towards as a preferred means of working with apt repositories. You can still use the version of node from debsource just use the apt commands instead of apt-get.

Reply:
Oh, OK, I see now.  Will fix.